src/directory/directory.mjs

```diff
@@ -78,6 +81,9 @@ export const directory = {
     {
       path: 'src/pages/gen2/index.tsx',
       children: [
+        {
+          path: 'src/pages/gen2/index.tsx'
+        },
```

Comment:
Added this "again" so that it shows up in the menu.

Some more info:
In `Layout.tsx`, we pass the parent node so that we can separate out `gen2` and the `[platform]` routes. Since `gen2` is a subpage, but it also needs to be it's own "home" page, I added it again so that it gets shown in the menu

If this is a